api/src/com/cloud/network/vpc/VpcProvisioningService.java

```diff
@@ -26,13 +25,11 @@
 
     public VpcOffering getVpcOffering(long vpcOfferingId);
 
-    public VpcOffering createVpcOffering(String name, String displayText, List<String> supportedServices,
-                                         Map<String, List<String>> serviceProviders,
-                                         Map serviceCapabilitystList,
-                                         Long serviceOfferingId);
+    public VpcOffering createVpcOffering(String name, String displayText, List<String> supportedServices, Map<String, List<String>> serviceProviders, Map serviceCapabilitystList,
+            Long serviceOfferingId);
```

Comment:
@jburwell 
Have picked up the formatter for eclipse from https://cwiki.apache.org/confluence/display/CLOUDSTACK/Coding+conventions
Have not been fixing the style and formatting by hand but relying on the auto formatting that conforms to apache standard as per the above wiki.